python/tvm/autotvm/tuner/graph_tuning.py

```diff
@@ -0,0 +1,85 @@
+"""Utilities for tuning a whole graph (a set of tasks)"""
+
+import os
+from . import callback, XGBTuner, GATuner, RandomTuner, GridSearchTuner
+from .. import record, task
+
+def tune_tasks(tasks,
+               measure_option,
+               tuner='xgb',
+               n_trial=500,
+               early_stopping=200,
+               log_filename='tuning.log',
+               use_transfer_learning=True,
+               try_winograd=True):
```

Comment:
should try_winograd be part of tuner?  This seems to be a bad design choice, imagine we want to also add try_fft, try_im2col etc. Maybe we should have a GraphPolicy to subsume this option?